db/migrate/20150203165922_create_followings.rb

```diff
@@ -0,0 +1,8 @@
+class CreateFollowings < ActiveRecord::Migration
+  def change
+    create_table :followings do |t|
+      t.timestamps null:false
+      t.integer :user_id, :follower_id, null:false
```

Comment:
Be careful putting the requirements at the db level. While this provides an extra layer of protection, it also introduces another level you need to remember to change if you decide to change how your app works.